aten/src/ATen/native/native_functions.yaml

```diff
@@ -3613,133 +3613,157 @@
   dispatch:
     CPU: ne_scalar_out_cpu
     CUDA: ne_scalar_out_cuda
+    QuantizedCPU: quantized_ne_out
```

Comment:
as we discussed, name should be `ne_scalar_out_quantized_cpu` in this case

Reply:
Even if can overload them?